docs/reference/api/drivers/ResetReason.md

```diff
@@ -0,0 +1,39 @@
+## ResetReason
+
+Use the ResetReason interface to access hardware reset reason registers in a portable fashion.
+
+### ResetReason class reference
+
+[![View code](https://www.mbed.com/embed/?type=library)](http://os-doc-builder.test.mbed.com/docs/v5.8/mbed-os-api-doxy/classmbed_1_1_resetreason.html)
+
+### ResetReason example
```

Comment:
Please add the examples from both files to the teams examples.

Reply:
What are the teams examples you're referring to?

Reply:
These: https://os.mbed.com/teams/mbed_example/

We've now given you access (we think), so please create your examples here, and let me know if you still need permissions.